variables.tf

```diff
@@ -21,8 +21,14 @@ variable "allowed_aws_services_for_sns_published" {
 
 variable "kms_master_key_id" {
   type        = string
-  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK"
-  default     = "alias/aws/sns"
+  description = "The ID of an AWS-managed customer master key (CMK) for Amazon SNS or a custom CMK. If left empty, SNS Topic will have Encryption disabled."
+  default     = ""
+}
+
+variable "use_default_aws_sns_kms_master_key_id" {
```

Comment:
you could make the name simpler since this is already a sns module so there is no need to add sns to the name.
so it could be `use_default_kms_master_key`

Reply:
Agree, I can make that happen.